src/NServiceBus.AcceptanceTests/Core/Sagas/When_adding_state_to_context.cs

```diff
@@ -1,15 +1,15 @@
-namespace NServiceBus.AcceptanceTests.Sagas
+namespace NServiceBus.AcceptanceTests.Core.Sagas
```

Comment:
All 3 tests where IMO testing core behavior only:

When_finder_cant_find_saga_instance.cs => checks that we start a new saga if finder won't find a existing instance

When_overriding_saga_id_creation => checks that we invoke custom id generators properly

When_adding_state_to_context => verifies that we pass the context properly to saga finders.